shared/test/middleware/helpers/test_storage_apps.rb

```diff
@@ -148,6 +148,28 @@ def test_set_content_moderation
     assert_equal false, storage_apps.content_moderation_disabled?(new_project_channel_id)
   end
 
+  def restore
```

Comment:
This method should probably start with 'test'?

Reply:
This was a great catch, because it turns out that this test wasn't run before without the `test_` prefix.